utils/paddle/parse-money.ts

```
@@ -0,0 +1,333 @@
+  let convertedAmount: number;
+
+  if (ZERO_DECIMAL_CURRENCIES.includes(normalizedCurrency as any)) {
+    // Zero-decimal currencies (JPY, KRW, etc.) - no division needed
+    convertedAmount = parsedAmount;
+  } else {
+    // Standard currencies (USD, EUR, etc.) - divide by 100
+    convertedAmount = parsedAmount / 100;
+  }
+
+  // Validate the result
+  if (isNaN(convertedAmount) || !isFinite(convertedAmount)) {
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)